test/dm_control_test.py

```diff
@@ -0,0 +1,118 @@
+"""Tests the functionality of the dm_env_wrapper on dm_control envs."""
+
+import numpy as np
+import pytest
+from dm_control import suite
+from dm_control.suite import (
+    acrobot,
+    ball_in_cup,
+    cartpole,
+    cheetah,
+    dog,
+    finger,
+    fish,
+    hopper,
+    humanoid,
+    humanoid_CMU,
+    lqr,
+    manipulator,
+    pendulum,
+    point_mass,
+    quadruped,
+    reacher,
+    stacker,
+    swimmer,
+    walker,
+)
+from gymnasium.utils.env_checker import check_env
+from PIL import Image
+
+from shimmy import dm_env_wrapper
+
+# Find all domains imported.
+_DOMAINS = [
+    acrobot,
+    ball_in_cup,
+    cartpole,
+    cheetah,
+    dog,
+    finger,
+    fish,
+    hopper,
+    humanoid,
+    humanoid_CMU,
+    lqr,
+    manipulator,
+    pendulum,
+    point_mass,
+    quadruped,
+    reacher,
+    stacker,
+    swimmer,
+    walker,
+]
+
+
+@pytest.mark.parametrize("domain", _DOMAINS)
+def test_all_envs(domain):
+    """Tests the conversion of all dm_control envs."""
+    # for each possible task in the domain:
+    for task in domain.SUITE.values():
+
+        # convert the task to gymnasium environment
+        env = dm_env_wrapper(task(), render_mode="rgb_array")
+
+        # check the environment using gymnasium
+        check_env(env)
+
+        # reset and begin test
+        env.reset()
+        term, trunc = False, False
+
+        # run until termination
+        while not term and not trunc:
```

Comment:
Should this not be OR?

Reply:
Nope, it can be either `not (term or trunc)`, or `not term and not trunc`.

Reply:
Nope, it's either `not (term or trunc)` or `not term and not trunc` which is what is done here.